src/app/settings/sponsored-families.component.html

```diff
@@ -17,16 +17,16 @@ <h1>{{'sponsoredFamilies' | i18n}}</h1>
         </ul>
     </div>
     <form #form (ngSubmit)="submit()" [appApiAction]="formPromise" ngNativeValidate *ngIf="anyOrgsAvailable">
-        <div *ngIf="moreThanOneOrgAvailable" class="form-group col-6">
-            <label for="availableSponsorshipOrg">{{ 'sponsoredFamiliesSelectOffer' | i18n}}</label>
```

Comment:
Are we going to still use this text in the https://github.com/bitwarden/web/blob/6ec22a9408acf188955c26ce6880600636e769cb/src/app/organizations/sponsorships/families-for-enterprise-setup.component.html ?

Reply:
We are, that's why the text didn't make sense before. We were inadvertently applying the same message to both the redeem page and the setup page, though they refer to different organziations